lib/linguist/heuristics.rb

```diff
@@ -226,6 +226,16 @@ def call(data)
       end
     end
 
+    disambiguate ".gml" do |data|
+      if /^\s*(\<\?xml|xmlns)/i.match(data)
+        Language["XML"]
+      elsif /^\s*((graph|node)+[\s\n\r]+\[$)/i.match(data)
```

Comment:
We don't need the outer parentheses here. Also, is the `+` after `(graph|node)` an error?

Reply:
Ah, you're right ^_^